<sdf version="1.4">
  <world name="default">
    <include>
      <uri>model://ground_plane</uri>
    </include>

    <model name="lampadina"> 
      <static>true</static> 
      <link name="link"> 
        <collision name="collision"> 
          <geometry> 
            <box> 
              <size>0.2 0.5 0.5</size>  
            </box> 
          </geometry> 
        </collision> 
        <visual name="visual"> 
          <geometry> 
            <box> 
              <size>0.2 0.5 0.5</size>  
            </box> 
          </geometry> 
          <material> 
            <script> 
              <uri>file://media/materials/scripts/gazebo.material</uri> 
              <name>Gazebo/Yellow</name>  
            </script> 
          </material> 
        </visual> 
      </link> 
      <pose>5 2.5 0 0 0 0</pose> 
    </model>


    <model name="lampadina_2"> 
      <static>true</static> 
      <link name="link"> 
        <collision name="collision"> 
          <geometry> 
            <box> 
              <size>0.2 0.5 0.5</size>  
            </box> 
          </geometry> 
        </collision> 
        <visual name="visual"> 
          <geometry> 
            <box> 
              <size>0.2 0.5 0.5</size>  
            </box> 
          </geometry> 
          <material> 
            <script> 
              <uri>file://media/materials/scripts/gazebo.material</uri> 
              <name>Gazebo/Yellow</name>  
            </script> 
          </material> 
        </visual> 
      </link> 
      <pose>5 3.5 0 0 0 0</pose>  
    </model>

    <model name="lampadina_3"> 
      <static>true</static> 
      <link name="link"> 
        <collision name="collision"> 
          <geometry> 
            <box> 
              <size>0.2 0.5 0.5</size>  
            </box> 
          </geometry> 
        </collision> 
        <visual name="visual"> 
          <geometry> 
            <box> 
              <size>0.2 0.5 0.5</size>  
            </box> 
          </geometry> 
          <material> 
            <script> 
              <uri>file://media/materials/scripts/gazebo.material</uri> 
              <name>Gazebo/Yellow</name>  
            </script> 
          </material> 
        </visual> 
      </link> 
      <pose>5 4.5 0 0 0 0</pose>  
    </model>


    
    <scene>
      <ambient>0.4 0.4 0.4 1</ambient>
      <background>0.7 0.7 0.7 1</background>
      <shadows>true</shadows>
    </scene>
  </world>
</sdf>